test/unit/nodes/element.js

```diff
@@ -18,6 +18,23 @@ suite('Element', function() {
 			);
 		});
 
+		suite('#querySelectorAll', function() {
+			test('css selector with tag name only', function() {
+				var lis = elem.querySelectorAll('li');
+
+				assert.equal(lis.length, 3);
+			});
+		});
+
+		suite('#querySelector', function() {
+			test('returns first matching node', function() {
+				var lis = elem.querySelectorAll('li');
+				var li = elem.querySelector('li');
+
+				assert.equal(li, lis[0]);
```

Comment:
Mind getting a reference to the expected element by manually traversing `elem` via `childNodes` properties? This will allow the two tests you've written to fail independently